au4/src/effects/builtin/general/GeneralEffectView.qml

```diff
@@ -0,0 +1,33 @@
+import QtQuick
+
+import Muse.UiComponents
+
+import Audacity.Effects
+
+Rectangle {
+
+    property string title: "Amplify"
```

Comment:
I suppose this property will also have to be read from `viewModel` ?

Reply:
yes, that's probably how it will be